### DISCUSSION TITLE
bump-cask-pr not working anymore

### DISCUSSION THREAD

**Discussion by ZepDonald** [Homebrew/discussions#758] [Casks]
I've started contributing some version changes for some casks I'm using a few weeks ago via `brew bump-cask-pr`. Last week I encountered a problem which I don't know how to fix.

Every time I use the command, it throws an error at the end:
``
! [remote rejected]       bump-webex-41.2.0.17887 -> bump-webex-41.2.0.17887 (refusing to allow an OAuth App to create or update workflow `.github/workflows/bump-unversioned-casks.yml` without `workflow` scope)
``

It looks like there's an issue in the mentioned file `bump-unversioned-casks.yml`.

Is this a problem with Homebrew or am I doing something wrong?

**Comment by SMillerDev** [comment]
I think you either need to extend the permissions of your github token, or you need to manually rebase the branch to make sure it includes the latest workflow changes.

**Reply by ZepDonald** [reply]
My GitHub token is correct. I used `brew bump-cask-pr` a few times during the last 4 weeks without any problems and I never changed my token settings. I also double checked the permissions and workflow is checked.

I also run the command 'brew update-reset' to rebase. From what I see in the man pages, this should be the right command, but I might be wrong.

I also compared my local `bump-unversioned-casks.yml`with the one from the repo and couldn't find any differences.

Any further ideas?

**Comment by VergeDX** [comment]
Same issues here: [pavlobu/deskreen](https://github.com/pavlobu/deskreen/runs/1942855611?check_suite_focus=true#step:4:210).
I found the possible solution of [StackOverflow](https://stackoverflow.com/questions/64059610/how-to-resolve-refusing-to-allow-an-oauth-app-to-create-or-update-workflow-on), but I have not tried.

**Comment by VergeDX** [comment]
See [Homebrew/brew/pull/10568](https://github.com/Homebrew/brew/pull/10568), now Homebrew need "gist, public_repo, workflow" permission.

**Comment by ZepDonald** [comment]
I got it working, but I'm more confused now than before.

I never set the environment variable HOMEBREW_GITHUB_API_TOKEN. Now I did and it's working like a charm.
I just don't know, why it was working before and what made it stop working.
